docs/en/siem/cases-api-get-case.asciidoc

```diff
@@ -0,0 +1,76 @@
+[[cases-api-get-case]]
+=== Get case
+
+Returns the specified case.
+
+==== Request URL
+
+`GET /api/cases/<case ID>`
+
+===== URL parts
+
+The URL must include the `case ID` of the case you are retrieving. Call
+<<cases-api-find-cases>> to retrieve case IDs.
+
+===== URL query parameters
+ 
+[width="100%",options="header"]
+|==============================================
+|Name |Type |Description |Required
+
+|`includeComments` |Boolean |Determines whether case comments are 
+returned. |No, defaults to `true`.
+|==============================================
+
+===== Example request
+
+Returns case ID `eb696730-66a2-11ea-be1b-2bd3fef48984` without comments:
+
+[source,sh]
+--------------------------------------------------
+GET api/cases/c2ffcea0-57d0-11ea-95da-cb9f4a010e31?includeComments=false
+--------------------------------------------------
+// KIBANA
+
+==== Response code
+
+`200`:: 
+   Indicates a successful call.
+
+==== Response payload
+
+The requested case JSON object.
+
+===== Response example
+
+[source,json]
+--------------------------------------------------
+{
+  "id": "eb696730-66a2-11ea-be1b-2bd3fef48984",
+  "version": "WzI0LDFd",
+  "comments": [],
+  "comment_ids": [
+    "35180680-66a7-11ea-be1b-2bd3fef48984"
+  ],
+  "created_at": "2020-03-15T09:54:10.654Z",
+  "created_by": {
+    "full_name": "Alan Hunley",
+    "username": "ahunley"
+  },
+  "updated_at": "2020-03-15T10:39:02.360Z",
+  "updated_by": {
+    "full_name": "Classified",
+    "username": "M"
+  },
+  "description": "James Bond clicked on a highly suspicious email banner
+  advertising cheap holidays for underpaid civil servants. Operation
+  bubblegum is active. Repeat - operation bubblegum is now active!",
+  "title": "This case will self-destruct in 5 seconds",
+  "status": "open",
+  "tags": [
+    "phishing",
+    "social engineering",
+    "bubblegum"
+  ]
+}
```

Comment:
example response:
```
example response:
  ```
{
  "id": "c9b99d00-6ea7-11ea-9515-01db8adf8abd",
  "version": "WzIzNCwxXQ==",
  "comments": [
    {
      "id": "2e39f4e0-6ea9-11ea-b1ba-4dcf38eaac81",
      "version": "WzIwNSwxXQ==",
      "comment": "lalala",
      "created_at": "2020-03-25T14:59:09.189Z",
      "created_by": {
        "email": "case.mcgoo@elastic.co",
        "full_name": "Case McGoo",
        "username": "casetester"
      },
      "pushed_at": null,
      "pushed_by": null,
      "updated_at": null,
      "updated_by": null
    },
    {
      "id": "242f5980-6eaa-11ea-b1ba-4dcf38eaac81",
      "version": "WzIzNiwxXQ==",
      "comment": "comment 3 hey hey",
      "created_at": "2020-03-25T15:06:01.837Z",
      "created_by": {
        "email": "case.mcgoo@elastic.co",
        "full_name": "Case McGoo",
        "username": "casetester"
      },
      "pushed_at": null,
      "pushed_by": null,
      "updated_at": "2020-03-25T15:06:49.064Z",
      "updated_by": {
        "full_name": "Case McGoo",
        "email": "case.mcgoo@elastic.co",
        "username": "casetester"
      }
    },
    {
      "id": "1fa20930-6eaa-11ea-b1ba-4dcf38eaac81",
      "version": "WzIzMCwxXQ==",
      "comment": "comment 2",
      "created_at": "2020-03-25T15:05:54.194Z",
      "created_by": {
        "email": "case.mcgoo@elastic.co",
        "full_name": "Case McGoo",
        "username": "casetester"
      },
      "pushed_at": null,
      "pushed_by": null,
      "updated_at": null,
      "updated_by": null
    }
  ],
  "totalComment": 0,
  "closed_at": null,
  "closed_by": null,
  "created_at": "2020-03-25T14:49:11.076Z",
  "created_by": {
    "email": "case.mcgoo@elastic.co",
    "full_name": "Case McGoo",
    "username": "casetester"
  },
  "external_service": {
    "external_title": "INC0010023",
    "pushed_by": {
      "full_name": "Case McGoo",
      "email": "case.mcgoo@elastic.co",
      "username": "casetester"
    },
    "external_url": "https://dev61254.service-now.com//nav_to.do?uri=incident.do?sys_id=f5355b81dbfb0010797085184b961917",
    "pushed_at": "2020-03-25T15:06:31.521Z",
    "connector_id": "46d2727c-96ad-4912-ac5e-273f553dbc7f",
    "external_id": "f5355b81dbfb0010797085184b961917",
    "connector_name": "Steph sn"
  },
  "updated_at": "2020-03-25T15:06:31.521Z",
  "updated_by": {
    "full_name": "Case McGoo",
    "email": "case.mcgoo@elastic.co",
    "username": "casetester"
  },
  "description": "This looks not so good",
  "title": "Bad meanie defacing data",
  "status": "open",
  "tags": [
    "defacement"
  ]
}
```
```